packages/wrangler/package.json

```diff
@@ -36,19 +36,18 @@
     "cli"
   ],
   "dependencies": {
-    "esbuild": "0.14.23",
+    "esbuild": "0.14.34",
     "miniflare": "2.4.0",
     "path-to-regexp": "^6.2.0",
-    "selfsigned": "^2.0.0",
+    "selfsigned": "^2.0.1",
     "semiver": "^1.1.0",
-    "timeago.js": "^4.0.2",
```

Comment:
@sidharthachatterjee  we bundle wrangler, so it made sense to move this into devDependencies